caramel/config.py

```diff
@@ -162,19 +162,21 @@ def get_db_url(arguments=None, settings=None, required=True):
 
 def get_log_level(argument_level, logger=None, env=None):
     """Calculates the highest verbosity(here inverted) from the argument,
-    environment and root, capping it to between 0-3, returning the log level"""
+    environment and root, capping it to between 10-40, returning the log level"""
 
     if env is None:
         env = os.environ
-    env_level = int(env.get("CARAMEL_LOG_LEVEL", 0))
+    env_level = LOG_LEVEL[env.get("CARAMEL_LOG_LEVEL", "ERROR").upper()]
 
     if logger is None:
         logger = logging.getLogger()
-    current_level = LOG_LEVEL.index(logger.level) if logger.level in LOG_LEVEL else 0
+    current_level = logger.level
 
-    verbosity = max(argument_level, env_level, current_level)
-    verbosity = min(verbosity, len(LOG_LEVEL) - 1)
-    log_level = LOG_LEVEL[verbosity]
+    argument_verbosity = 40 - argument_level * 10
```

Comment:
40 and 10 are magic numbers and could definitely need names. I happen to know that 40 is the same as "logging.ERROR" , but that might be confusing when I forget that.

Reply:
Very good point!